packages/shared/src/graphql/sources.ts

```diff
@@ -17,37 +17,6 @@ export interface Source {
 
 export type SourceData = { source: Source };
 
```

Comment:
@sshanzel this refactor is mostly unrelated to the PR. I recommend separating big tasks like this in a separate PR in the future. It is ok for this PR though.

Reply:
I was having some dependencies cycle issue which is why I had to move some of them. Then I realized why not move everything 😅 

Will try to separate the PRs next time. Thanks for the review!